packages/web/src/components/Prizes/PrizesList.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Prize } from '@hangar/shared';
+import { Flex, Heading, ListItem, Text, UnorderedList } from '@chakra-ui/react';
+import { colors } from '../../theme/colors';
+
+type PrizeCardProps = {
+  prize: Prize;
+};
+
+type PrizesListProps = {
+  prizes: Prize[];
+};
+
+const RANKING= [ '🥇', '🥈', '🥉' ];
+const BONUS = '✨';
+
+const PrizeCard: React.FC<PrizeCardProps> = ({ prize }) => {
+  const { name, description, position, isBonus } = prize;
```

Comment:
position will be in [0, 1, 2] correct?

Reply:
it could be any positive int as far I can tell, but it is only used here when -1>position<3 per line 27